metadata/ch.qos.logback/logback-classic/index.json

```diff
@@ -1,14 +1,23 @@
 [
   {
     "latest": true,
+    "metadata-version": "1.4.9",
+    "module": "ch.qos.logback:logback-classic",
+    "tested-versions": [
+      "1.4.9"
+    ]
+  },
+  {
     "metadata-version": "1.4.1",
+    "default-for": "1\\.4\\.[0-8]",
```

Comment:
I wonder if we should say `[0-9]+`.

Reply:
`1.4.9` release has its own metadata (entry above), so this entry is for releases from `1.4.0` to `1.4.8`

Reply:
But I will need to change it to `1\\.4\\.[0-8]$` since we release `1.4.11` should not used `1.4.1`metadata

Reply:
Actually it already works as expected:
- releases from `1.4.0` to `1.4.8` are using `1.4.1` metadata
- `1.4.9` release and all above are using `1.4.9` metadata